src/module/nft/event.rs

```diff
@@ -0,0 +1,104 @@
+use std::str;
+
+use anyhow::Result;
+use plain::Plain;
+
+use crate::{
+    core::events::{
+        bpf::{parse_single_raw_section, BpfRawSection},
+        *,
+    },
+    event_section, event_section_factory,
+    module::ModuleId,
+};
+
+// Please keep in sync with its bpf counterpart under
+// src/modules/nft/bpf/nft.bpf.c
+const NFT_NAME_SIZE: usize = 128;
+
+struct NftName([u8; NFT_NAME_SIZE]);
+
+impl Default for NftName {
+    fn default() -> Self {
+        NftName([0; NFT_NAME_SIZE])
+    }
+}
+
+impl NftName {
+    fn to_string(&self) -> Result<String> {
+        Ok(str::from_utf8(&self.0)?
+            .trim_end_matches(char::from(0))
+            .into())
+    }
+
+    fn to_string_opt(&self) -> Result<Option<String>> {
+        let res = self.to_string()?;
+
+        if res.is_empty() {
+            return Ok(None);
+        }
+
+        Ok(Some(res))
+    }
+}
+
+/// Nft event section
+#[event_section]
+pub(crate) struct NftEvent {
+    table_name: Option<String>,
+    chain_name: Option<String>,
+    verdict: Option<String>,
+    verdict_chain_name: Option<String>,
+}
+
+/// Nft info.
+#[derive(Default)]
+#[repr(C, packed)]
+struct NftEvt {
+    /// Table name.
+    tn: NftName,
+    /// Chain name.
+    cn: NftName,
+    /// Verdict.
+    v: i32,
+    /// Verdict chain name.
+    vcn: NftName,
+}
+
+unsafe impl Plain for NftEvt {}
+
+#[derive(Default)]
+#[event_section_factory(NftEvent)]
+pub(crate) struct NftEventFactory {}
+
+impl RawEventSectionFactory for NftEventFactory {
+    fn from_raw(&mut self, raw_sections: Vec<BpfRawSection>) -> Result<Box<dyn EventSection>> {
+        let mut event = NftEvent::default();
+        let raw = parse_single_raw_section::<NftEvt>(ModuleId::Nft, raw_sections)?;
+
+        event.table_name = raw.tn.to_string_opt()?;
+        event.chain_name = raw.cn.to_string_opt()?;
+        event.verdict_chain_name = raw.vcn.to_string_opt()?;
+
+        event.verdict = Some(
+            match raw.v {
+                -1 => "CONTINUE",
+                -2 => "BREAK",
+                -3 => "JUMP",
+                -4 => "GOTO",
+                -5 => "RETURN",
+                0 => "DROP",
+                1 => "ACCEPT",
+                2 => "STOLEN",
+                3 => "QUEUE",
+                4 => "REPEAT",
+                /* NF_STOP is deprecated. */
+                5 => "STOP",
+                _ => "UNKNOWN",
```

Comment:
A small note, just to conveniently keep track of it in the discussion, in terms of actual values: all are stable as they are uapi and so we're ok in that sense.
In the case of added values, well, yes, it's an interesting suggestion. I'm not sure that will grow in a way to become a maintenance issue. To date, I kept it like this for consistency with the of NF_* counterpart and because we can only parse enums and this list is not just enum (and have half manual with half parsed).
Not 100% sure as well. I guess it could be done as a follow-up as well if we don't find strong reasons now to include it (and maybe change our mind later). WDYT?